modules/codegen/src/smithy4s/codegen/internals/CodegenImpl.scala

```diff
@@ -175,15 +175,17 @@ private[codegen] object CodegenImpl { self =>
       args.specs.map(_.toIO).toSet,
       args.dependencies,
       args.repositories,
-      withAwsTypeTransformer(args.transformers),
+      withMandatoryTransformers(args.transformers),
       discoverModels = false,
       args.localJars
     )
 
     Node.prettyPrintJson(ModelSerializer.builder().build.serialize(model))
   }
 
-  private def withAwsTypeTransformer(transformers: List[String]): List[String] =
-    transformers :+ AwsConstraintsRemover.name :+ AwsStandardTypesTransformer.name :+ OpenEnumTransformer.name
+  private def withMandatoryTransformers(
+      transformers: List[String]
+  ): List[String] =
+    transformers :+ AwsConstraintsRemover.name :+ AwsStandardTypesTransformer.name :+ OpenEnumTransformer.name :+ ValidatedNewtypesTransformer.name
```

Comment:
**nitpick:** I'd split this as

```suggestion
    transformers :+ 
      AwsConstraintsRemover.name :+ 
      AwsStandardTypesTransformer.name :+ 
      OpenEnumTransformer.name :+ 
      ValidatedNewtypesTransformer.name
```

so that future additions don't cause a diff for the whole thing